client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -90,10 +90,11 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 		[ staticDesigns ]
 	);
 
-	const generatedDesigns = useGeneratedDesigns();
+	const { data: generatedDesigns, isLoading: isLoadingGeneratedDesigns } =
+		useGeneratedDesignsQuery();
 	const shuffledGeneratedDesigns = useMemo(
-		() => shuffle( generatedDesigns ),
-		[ generatedDesigns ]
+		() => ( ! isLoadingGeneratedDesigns ? shuffle( generatedDesigns ) : [] ),
```

Comment:
I think we can revert these changes by simply make `[]` as the default value of `generatedDesigns` above:

```
const { data: generatedDesigns = [], ...
```

(because `shuffle([])` is still `[]`)

Reply:
Good point 👍